Dashboard/app/js/lib/views/reports/export-reports-views.js

```diff
@@ -307,15 +307,17 @@ FLOW.ReportListItemView = FLOW.View.extend({
     var form  = FLOW.Survey.find(formId);
     if (form) {
       var ancestorIds = form.get('ancestorIds');
-      for (var i = 0; i < ancestorIds.length; i++) {
-        if (ancestorIds[i] !== null && ancestorIds[i] !== 0) {
-          var ancestor = FLOW.SurveyGroup.find(ancestorIds[i]);
-          if (ancestor) {
-            path += (i > 1 ? " > ": "")+ancestor.get('name');
+      if (ancestorIds) {
+        for (var i = 0; i < ancestorIds.length; i++) {
+          if (ancestorIds[i] !== null && ancestorIds[i] !== 0) {
+            var ancestor = FLOW.SurveyGroup.find(ancestorIds[i]);
+            if (ancestor) {
+              path += (i > 1 ? " > ": "")+ancestor.get('name');
+            }
           }
         }
+        path += " > "+form.get('name');
```

Comment:
same formatting issue as above? I am not familiar with js though, so maybe wrong